openshift/release/manifest-patches/005-secret-informer.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/openshift/release/artifacts/2-serving-core.yaml b/openshift/release/artifacts/2-serving-core.yaml
+index abe895e01..a66f4b233 100644
+--- a/openshift/release/artifacts/2-serving-core.yaml
++++ b/openshift/release/artifacts/2-serving-core.yaml
+@@ -4482,6 +4482,7 @@ metadata:
+   namespace: knative-serving
+   labels:
+     serving-certs-ctrl: "data-plane"
++    networking.internal.knative.dev/certificate-uid: "data-plane"
```

Comment:
Should we add this label via transformer code in SO?

Reply:
We can do that too, the reason this is added here is because it does no harm. I will add it here: https://github.com/openshift-knative/serverless-operator/pull/1764.

Reply:
Adding some patches makes sense but SO can change the manifest so I think we should do that.
Otherwise, we don't have the consistency and make confusion.